server.py

```diff
@@ -72,6 +88,25 @@ def parse_request(request):
         return meth[1]
 
 
+def resolve_uri(parse):
+    root = os.path.join(os.getcwd(), 'webroot')
```

Comment:
To reiterate what we mentioned in class, this means that webroot must be relative to the location where the server process is executed, which may or may not be the same thing as where the server code is located.